skytap/resource_skytap_vm_hardware_test.go

```diff
@@ -0,0 +1,315 @@
+package skytap
```

Comment:
should be part of the VM acceptance test probably, as is the same resource

Reply:
ok - will combine this file with the previous one - will do in #18

Reply:
done